### PR TITLE
Adds text instructions; fixes #371

### LINKED ISSUE #371: Add Apostille/Hand Signature request capability
Add Apostille/Hand Signature request capability.

Will likely live on the Review Record Request page.

Draft text from Registry:
"If you are requesting a certificate for International use that requires an Apostille from the MA Secretary of State you will require a HAND SIGNATURE by the Registry, if so please email birth@boston.gov with the name of the person on the record and their date of birth."

### PR DESCRIPTION
**Pull request by jessicamarcus**
Also adds some utility CSS to react-fleet.

### CODE REVIEW

File: modules-js/react-fleet/src/utilities/css.ts [hunk 1]
```diff
@@ -9,5 +9,19 @@ export const VISUALLYHIDDEN = css({
   overflow: 'hidden',
   padding: 0,
   position: 'absolute',
-  width: '1px'
+  width: '1px',
 });
+
+export const CLEARDEFAULTSTYLING = {
```
Comment (fionawhim) [inline modules-js/react-fleet/src/utilities/css.ts]: `CLEAR_DEFAULT_STYLING`?

File: services-js/registry-certs/client/birth/components/DisclosureComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
+import React, { useEffect, useState } from 'react';
+
+import { css } from 'emotion';
+
+import {
+  CLEARDEFAULTSTYLING,
+  OPTIMISTIC_BLUE_DARK,
+} from '@cityofboston/react-fleet';
+
+export default function DisclosureComponent(): JSX.Element {
```
Comment (fionawhim) [inline services-js/registry-certs/client/birth/components/DisclosureComponent.tsx]: This should probably be named more specifically to apostille messaging?

File: services-js/registry-certs/client/birth/components/DisclosureComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    </>
+  );
+}
+
+function detailsContent(): React.ReactChild {
+  return (
+    <>
+      <p>
+        You need to have a hand signature from the Registry. After you finish
+        your order, please email birth@boston.gov with:
```
Comment (fionawhim) [inline services-js/registry-certs/client/birth/components/DisclosureComponent.tsx]: Do you want to link this to the contact form?

File: services-js/registry-certs/client/birth/components/DisclosureComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    cursor: 'pointer',
+
+    '> summary': {
+      color: OPTIMISTIC_BLUE_DARK,
+      textDecoration: 'underline',
+    },
+  },
+
+  // We rotate the path rather than the entire svg because the focus ring of the
+  // summary component gets visually pushed out as the svg rotates in Chrome.
```
Comment (fionawhim) [inline services-js/registry-certs/client/birth/components/DisclosureComponent.tsx]: nice touch

**Review by fionawhim — CHANGES_REQUESTED** [review]

File: services-js/registry-certs/client/birth/components/DisclosureComponent.tsx (hunk 3 above)
Comment (jessicamarcus) [reply services-js/registry-certs/client/birth/components/DisclosureComponent.tsx]: Nah - Patty doesn’t want to make this all that easy (so people aren’t requesting signatures for no actual reason). Also, the user is supposed to send the email after the order was placed. There’s got to be a smoother process while satisfying her concerns about excessive requests, but this works for now.

File: services-js/registry-certs/client/birth/components/DisclosureComponent.tsx (hunk 2 above)
Comment (jessicamarcus) [reply services-js/registry-certs/client/birth/components/DisclosureComponent.tsx]: Duh - thx!

**Review by fionawhim — APPROVED** [review]

**Comment by fionawhim** [comment]
LGTM!

**Pull request merged**